ansible/roles/installation/nvidia-docker/tasks/main.yml

```diff
@@ -28,3 +28,47 @@
   when: nvidia_docker | bool and not skip_nvidia|bool
   become: yes
   changed_when: false
+
+- name: Check what graphic drivers are used
+  block:
+  - name: install nvidia-prime apt
+    apt:
+      name: nvidia-prime
+      state: present
+    become: yes
+
+  - name: query prime-select
+    command: prime-select query
+    become: yes
+    register: prime_select_results
+
+  - name: switch driver to nvidia
+    command: prime-select nvidia
+    become: yes
+    when: "prime_select_results.stdout != 'nvidia'"
+
+  - name: reboot the computer
+    debug:
+      msg: You used graphic driver has been switched to nvidia. Please reboot your PC and rerun the oneliner.
+    when: "prime_select_results.stdout != 'nvidia'"
+
+  - name: end playbook so user can reboot
+    meta: end_play
+    when: "prime_select_results.stdout != 'nvidia'"
```

Comment:
The reason we want to make sure its nvidia is because it has previously caused issues with teleop. So its best to enforce nvidia as theres no real reason not to use it